src/site/content/en/blog/fast-playback-with-preload/index.md

```diff
@@ -0,0 +1,604 @@
+---
+layout: post
+title: Fast playback with audio and video preload
+subhead: How to accelerate your media playback by actively preloading resources.
+authors:
+  - beaufortfrancois
+description: |
+  Faster playback start means more people watching your video or listening to
+  your audio. That's a known fact. In this article I'll explore techniques you
+  can use to accelerate your media playback by actively preloading resources
+  depending on your use case.
+date: 2017-08-17
+updated: 2020-11-03
+tags:
+  - media
+  - video
+  - performance
+  - network
+---
+
+Faster playback start means more people watching your video or listening to your
+audio. [That's a known fact](https://www.digitaltrends.com/web/buffer-rage/). In
+this article I'll explore techniques you can use to accelerate your audio and
+video playback by actively preloading resources depending on your use case.
+
+<figure>
+  <video controls controlsList="nodownload" muted playsinline style="width: 100%">
+    <source src="https://storage.googleapis.com/webfundamentals-assets/videos/video-preload-hero.webm#t=1.1"
```

Comment:
Shall we move those video assets to web.dev storage bucket as well?

Reply:
Done.